pegasus/migrations/106_add_popularityrank_to_tutorials.rb

```diff
@@ -0,0 +1,7 @@
+Sequel.migration do
+  change do
+    alter_table(:tutorials) do
+      add_column :popularityrank, Integer, null: false
```

Comment:
Most of the gsheets are handled using a newer Pegasus system (v3) which does take care of creating the tables.  Unfortunately HocTutorials is one of the few using the original system, and we need to create the migrations for those.